code/mathematical_algorithms/src/replace_0_with_5/replace_0_with_5.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+
+namespace Replace
+{
+    public class Program
+    {
+        private static string replace(string n)
+        {
+            return n.Replace('0', '5');
```

Comment:
😑
Please implement the algorithm. Otherwise:

1. The function wrapper is pointless.
2. The algorithm doesn't even benefit anyone.

Reply:
ohk boss i will :P